src/standard/BaseStandardVerifier.sol

```diff
@@ -178,29 +185,32 @@ abstract contract BaseStandardVerifier {
                 // We add 0x24 to skip the selector + the length of the proof
                 let data_ptr := add(calldataload(0x04), 0x24)
 
-                mstore(W1_X_LOC, mod(calldataload(add(data_ptr, 0x20)), q))
+                // @follow-up Should we use the same naming as in the paper to reduce confusion? Seems like a no-brainer.
+                // @note The proof is stored differently than the paper with the following order:
+                // [a]1, [b]1, [c]1, [z]₁, [tlo]1, [tmid]1, [thi]1, a̅, b̅, c̅, s̅_σ₁, s̅_σ₂, z̅_ω,  [Wz]1, [Wzω]1
```

Comment:
More the naming, in the paper we use a,b,c in code w1,w2,w3. Zac elaborated on it being useful to use w_index as we could have higher width so it is a way of not running out of letters. But just confusing when matching against paper.

Reply:
Yeah, it's annoying. In the paper we use $a$, $b$, $c$ for the blinded versions of $w_1$, $w_2$, $w_3$, but it would be  better to use a more generic notation for blinding so we can allow the width to vary.